src/components/map/MapSearch.js

```diff
@@ -184,7 +182,7 @@ const MapSearch = () => {
       return 'Force is not-physical';
     }
   };
-console.log(lastIncident)
+  console.log(lastIncident);
```

Comment:
Remove console logs from code going to production.

Reply:
Will do